assignments/this.js

```diff
@@ -1,25 +1,48 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Global
```

Comment:
I would definitely add the written definition for each principle. I'm almost positive it will be on your test.